docs/services_03_transactions.md

```diff
@@ -0,0 +1,218 @@
+---
+id: services_transactions
+title: Transactions Service
+sidebar_label: Transactions Service
+---
+
+## Transaction Service
+Keeps track of transactions sent via Gnosis Safe contracts. It uses events and [tracing](https://wiki.parity.io/JSONRPC-trace-module).
+
+Transactions are detected in an automatic way, so there is no need of informing the service about the transactions as in
+previous versions of the *Transaction Service*
+
+- [GitHub](https://github.com/gnosis/safe-transaction-service)
+
+- [Releases](https://github.com/gnosis/safe-transaction-service/releases)
+
+- [Swagger (Mainnet version)](https://safe-transaction.gnosis.io/)
+
+- [Swagger (Rinkeby version)](https://safe-transaction.rinkeby.gnosis.io/)
+
+- [Safe Contracts and addresses on networks](https://github.com/gnosis/safe-contracts/releases)
```

Comment:
shouldn't this go into the contracts section?

Reply:
Ignored for now until we rework the services section